library/swipl

```diff
@@ -1,11 +1,11 @@
 Maintainers: Jan Wielemaker <jan@swi-prolog.org> (@JanWielemaker),
              Dave Curylo <dave@curylo.org> (@ninjarobot)
 GitRepo: https://github.com/SWI-Prolog/docker-swipl.git
-GitCommit: 3ef22f678af652268d24285defc0ae60e62e6d26
+GitCommit: e5e2a61337a7f21d8b6c53fb6521d507b9d48771
```

Comment:
It seems this commit does not exist in the SWI-Prolog/docker-swipl repository. This gives a 404: https://github.com/SWI-Prolog/docker-swipl/commit/e5e2a61337a7f21d8b6c53fb6521d507b9d48771

Reply:
> It seems this commit does not exist in the SWI-Prolog/docker-swipl repository

Thanks.   Yes, forgot to push :cry:    Can you restart the checks or should I delete this PR and recreate exactly the same?

Reply:
The jobs have been restarted.